node_test.go

```diff
@@ -0,0 +1,96 @@
+package iavl
+
+import (
+	"bytes"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNode_aminoSize(t *testing.T) {
+	node := &Node{
+		key:       randBytes(10),
+		value:     randBytes(10),
+		version:   1,
+		height:    0,
+		size:      100,
+		hash:      randBytes(20),
+		leftHash:  randBytes(20),
+		leftNode:  nil,
+		rightHash: randBytes(20),
+		rightNode: nil,
+		persisted: false,
+	}
+
+	// leaf node
+	var buf bytes.Buffer
+	err := node.writeBytes(&buf)
+	require.NoError(t, err)
+	require.Equal(t, buf.Len(), node.aminoSize())
+
+	// non-leaf node
+	node.height = 1
+	buf.Reset()
+	err = node.writeBytes(&buf)
+	require.NoError(t, err)
+	require.Equal(t, buf.Len(), node.aminoSize())
+}
+
+func BenchmarkNode_aminoSize(b *testing.B) {
```

Comment:
Can you pipe benchmark results here to a file and save in `benchmarks/results/169/`?

Reply:
added